src/main/java/com/kodilla/ecommercee/GroupController.java

```diff
@@ -1,4 +1,55 @@
 package com.kodilla.ecommercee;
 
+import com.kodilla.ecommercee.domain.Group;
+import com.kodilla.ecommercee.domain.GroupDto;
+import com.kodilla.ecommercee.mapper.GroupMapper;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.springframework.http.MediaType.APPLICATION_JSON_VALUE;
+
+@RestController
+@RequestMapping("/groups")
 public class GroupController {
+
+    @Autowired
+    private GroupMapper groupMapper;
```

Comment:
wstrzykiwanie przez pole jest ok, ale lepszym rozwiazaniem byloby wstrzykiwanie przez konstrutor
https://docs.spring.io/spring-boot/docs/current/reference/html/using-boot-spring-beans-and-dependency-injection.html